.prettierignore

```diff
@@ -9,3 +12,7 @@ node_modules
 /postgres-data
 
 /app/styles/tailwind.css
+
+*.md
```

Comment:
Let's include `.md` in Prettier, it will happily format that for us too.

Reply:
Alright, I just remember having issues with prettier formatting my .md files and breaking it. The issue was on my side, I'm just not quite sure what the main cause was